compiler/src/dmd/mars.d

```diff
@@ -901,11 +901,6 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
             target.isX86_64 = false;
             target.omfobj = false;
         }
-        else if (arg == "-m32omf") // https://dlang.org/dmd.html#switch-m32omfobj
```

Comment:
As long as we keep the code generation logic, perhaps this could stay as an undocumented switch?

Reply:
The idea is to kill off the rest of it once this pulled in stages.

Reply:
We still have to deal with the installer so a release can happen.

Cleaning the rest of the OMF support can wait, it's going to be a bit to untangle I expect.